app/controllers/step_by_step_pages_controller.rb

```diff
@@ -78,7 +78,11 @@ def publish
 
   def schedule
     set_current_page_as_step_by_step
-    if request.post?
+  end
+
+  def schedule_datetime
```

Comment:
This method is quite long and difficult to read. I wonder if it could be split out into well named methods that document what it's doing? Something like:

```ruby
def schedule
  set_current_page_as_step_by_step
  if !changenotes_supplied?
    render :schedule_changenotes
  elsif !datetime_inputs_supplied? || parse_datetime_inputs.issues.any?
    render :schedule_datetime
  else
    schedule!(changenotes, datetime)
  end
end
```

Reply:
Indeed, this method does need breaking up (a particularly prime candidate would be the parsing code).  I've chosen to just add the changes needed here to deal with the change note parameters, leaving the rest of the method untouched.  We can do a refactor later on, if there's time at the end of the mission.